openstack/cloudeyeservice/v2/alarmrule/results.go

```diff
@@ -0,0 +1,101 @@
+package alarmrule
+
+import (
+	"fmt"
+
+	"github.com/chnsz/golangsdk"
+)
+
+type CreateResponse struct {
+	AlarmID string `json:"alarm_id"`
+}
+
+type CreateResult struct {
+	golangsdk.Result
+}
+
+func (c CreateResult) Extract() (*CreateResponse, error) {
+	r := &CreateResponse{}
+	return r, c.ExtractInto(r)
+}
+
+type ResourcesInfo struct {
+	ResourceGroupID   string          `json:"resource_group_id"`
+	ResourceGroupName string          `json:"resource_group_name"`
+	Dimensions        []DimensionOpts `json:"dimensions"`
+}
+
+type AlarmRule struct {
+	AlarmID               string             `json:"alarm_id"`
+	Name                  string             `json:"name"`
+	Description           string             `json:"description"`
+	Namespace             string             `json:"namespace"`
+	ResourceGroupID       string             `json:"resource_group_id"`
```

Comment:
it seems that `resource_group_id` is not in the response body

Reply:
fixed